phpunit/blocks/render-query-test.php

```diff
@@ -0,0 +1,228 @@
+<?php
+/**
+ * Tests for the Query block rendering.
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ * @since 6.0.0
+ *
+ * @group blocks
+ */
+class Tests_Blocks_RenderQueryBlock extends WP_UnitTestCase {
+
+	private static $post_1;
+	private static $post_2;
+	private static $post_3;
+
+	public function set_up() {
+		parent::set_up();
+
+		self::$post_1 = self::factory()->post->create_and_get(
+			array(
+				'post_type'    => 'post',
+				'post_status'  => 'publish',
+				'post_name'    => 'post-1',
+				'post_title'   => 'Post 1',
+				'post_content' => 'Post 1 content',
+				'post_excerpt' => 'Post 1',
+			)
+		);
+
+		self::$post_2 = self::factory()->post->create_and_get(
+			array(
+				'post_type'    => 'post',
+				'post_status'  => 'publish',
+				'post_name'    => 'post-2',
+				'post_title'   => 'Post 2',
+				'post_content' => 'Post 2 content',
+				'post_excerpt' => 'Post 2',
+			)
+		);
+
+		self::$post_3 = self::factory()->post->create_and_get(
+			array(
+				'post_type'    => 'post',
+				'post_status'  => 'publish',
+				'post_name'    => 'post-2',
+				'post_title'   => 'Post 2',
+				'post_content' => 'Post 2 content',
+				'post_excerpt' => 'Post 2',
+			)
+		);
+
+		register_block_type(
+			'test/plugin-block',
+			array(
+				'render_callback' => static function () {
+					return '<div class="wp-block-test/plugin-block">Test</div>';
+				},
+			)
+		);
+	}
+
+	public function tear_down() {
+		parent::tear_down();
+		unregister_block_type( 'test/plugin-block' );
```

Comment:
Parent teardown ought to follow local teardown

```suggestion
		unregister_block_type( 'test/plugin-block' );
		parent::tear_down();
```

Reply:
Done in https://github.com/WordPress/gutenberg/pull/55714/commits/f05ffe9f0286a206628345f43f17e2a7328aa760.